react-app/src/hooks/mutations/useDeleteJaneRecord.ts

```
@@ -0,0 +1,16 @@
+import { deleteJaneByIds } from "@/backend/FirestoreCalls";
+import { JaneID } from "@/types/JaneType";
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+
+export function useDeleteJaneRecord() {
+  return useMutation({
+    mutationFn: async (rows: JaneID[]) => {
+      const ids = rows.map((entry) => entry.id);
+      await deleteJaneByIds(ids);
+    },
+    onSuccess: () => {
+      const queryClient = useQueryClient();
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)